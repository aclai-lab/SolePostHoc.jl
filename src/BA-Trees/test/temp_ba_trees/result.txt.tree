DATASET_NAME: dataset.train.csv
ENSEMBLE: BA
NB_TREES: 1
NB_FEATURES: 4
NB_CLASSES: 3
MAX_TREE_DEPTH: 8
Format: node / node type(LN - leave node, IN - internal node) left child / right child / feature / threshold / node_depth / majority class (starts with index 0)

[TREE 0]
NB_NODES: 65
0 IN 1 12 1 2.422 0 -1
1 IN 2 9 2 4.268 1 -1
2 IN 3 8 3 0.304 2 -1
3 IN 4 7 2 3.455 3 -1
4 IN 5 6 0 7.222 4 -1
5 LN -1 -1 -1 -1 5 0
6 LN -1 -1 -1 -1 5 1
7 LN -1 -1 -1 -1 4 1
8 LN -1 -1 -1 -1 3 1
9 IN 10 11 0 5.912 2 -1
10 LN -1 -1 -1 -1 3 1
11 LN -1 -1 -1 -1 3 2
12 IN 13 22 3 0.304 1 -1
13 IN 14 19 0 7.222 2 -1
14 IN 15 16 2 5.932 3 -1
15 LN -1 -1 -1 -1 4 0
16 IN 17 18 1 3.032 4 -1
17 LN -1 -1 -1 -1 5 0
18 LN -1 -1 -1 -1 5 2
19 IN 20 21 2 3.455 3 -1
20 LN -1 -1 -1 -1 4 0
21 LN -1 -1 -1 -1 4 2
22 IN 23 44 2 3.455 2 -1
23 IN 24 29 3 0.434 3 -1
24 IN 25 28 1 3.032 4 -1
25 IN 26 27 3 0.385 5 -1
26 LN -1 -1 -1 -1 6 0
27 LN -1 -1 -1 -1 6 2
28 LN -1 -1 -1 -1 5 0
29 IN 30 37 0 4.614 4 -1
30 IN 31 36 1 3.193 5 -1
31 IN 32 35 3 2.067 6 -1
32 IN 33 34 1 3.032 7 -1
33 LN -1 -1 -1 -1 8 2
34 LN -1 -1 -1 -1 8 0
35 LN -1 -1 -1 -1 7 2
36 LN -1 -1 -1 -1 6 0
37 IN 38 39 1 3.193 5 -1
38 LN -1 -1 -1 -1 6 1
39 IN 40 43 0 6.958 6 -1
40 IN 41 42 3 2.067 7 -1
41 LN -1 -1 -1 -1 8 0
42 LN -1 -1 -1 -1 8 2
43 LN -1 -1 -1 -1 7 1
44 IN 45 56 0 5.912 3 -1
45 IN 46 49 1 2.741 4 -1
46 IN 47 48 2 4.268 5 -1
47 LN -1 -1 -1 -1 6 1
48 LN -1 -1 -1 -1 6 2
49 IN 50 55 2 3.501 5 -1
50 IN 51 52 1 3.193 6 -1
51 LN -1 -1 -1 -1 7 2
52 IN 53 54 3 2.067 7 -1
53 LN -1 -1 -1 -1 8 0
54 LN -1 -1 -1 -1 8 2
55 LN -1 -1 -1 -1 6 2
56 IN 57 64 2 3.501 4 -1
57 IN 58 59 1 3.193 5 -1
58 LN -1 -1 -1 -1 6 2
59 IN 60 63 0 6.958 6 -1
60 IN 61 62 3 2.067 7 -1
61 LN -1 -1 -1 -1 8 0
62 LN -1 -1 -1 -1 8 2
63 LN -1 -1 -1 -1 7 2
64 LN -1 -1 -1 -1 5 2
