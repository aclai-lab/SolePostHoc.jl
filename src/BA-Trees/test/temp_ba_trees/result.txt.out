dataset.train.csv,RF,4,4,3,4,8,32,33,1,0.00898,140729924852032,0,0
